services/mailer/mail_test.go

```diff
@@ -51,13 +51,13 @@ func TestComposeIssueCommentMessage(t *testing.T) {
 	InitMailRender(email)
 
 	tos := []string{"test@gitea.com", "test2@gitea.com"}
-	msg := composeIssueCommentMessage(issue, doer, "test body", comment, mailIssueComment, tos, "issue comment")
+	msg := composeIssueCommentMessage(issue, doer, models.ActionCommentIssue, false, "test body", comment, mailIssueComment, tos, "issue comment")
 
 	subject := msg.GetHeader("Subject")
 	inreplyTo := msg.GetHeader("In-Reply-To")
 	references := msg.GetHeader("References")
 
-	assert.Equal(t, subject[0], "Re: "+mailSubject(issue), "Comment reply subject should contain Re:")
+	assert.Equal(t, subject[0], "Re: "+defaultMailSubject(issue), "Comment reply subject should contain Re:")
```

Comment:
Similar, since we aren't using this default these aren't testing anything real. For example the templated included in this PR does not include RE in a reply but this test will pass because it isn't checking that (and nothing real is actually being tested here)

Reply:
@mrsdizzie The test has no template configured (as a result of being a unit test vs. an integration test), so it always uses the fallback function (`fallbackMailSubject()`, previously `defaultMailSubject()`). The `"Re: "` string will be included.